### ISSUE TITLE
Option to update geocoordinate should only use person selected by filter

### ISSUE THREAD

**Issue by bernardsilenou** [hzi-braunschweig/SORMAS-Project#4912]
### Bug Description
Issue 3545 added the feature to update the coordinates
### Steps to Reproduce
1. Connect to https://test1401.sormas-oegd.de/sormas-ui/
2. filter for person with any of the name having name peter. I had just 2 persons in my scenario. 
3. Give an address to a case a peter with valid but wrong coordinates. Ignore the warning and save the person form.
4. return to person form and set missing geo coordinate
5. sytem took long and returned error
5. This may be because all persons in the db were considered (since the user was a national user) leading to a timeout OR logic breaks when the initial coordinates were wrong.

### Expected Behavior
-  Use only persons retained by users filter 
- change the warning message accordingly by removing the part that say all persons that the user has access to.
### Screenshots
![image](https://user-images.githubusercontent.com/12996724/112699041-45592300-8e8b-11eb-909b-73abc689a1e8.png)

### System Details
* Device:
* SORMAS version:
* Android version/Browser:
* Server URL:
* User Role:

**Comment by DavidBaldsiefen** [comment]
Did you set legitimate addresses for the persons? As noted in the popup, the feature does currently ignore filters, as I don't know any situation where you'd only want to do this for a specific set of persons

**Comment by MateStrysewske** [comment]
@bernardsilenou If there are error messages like these, we always need error logs to be able to investigate the issue.

**Comment by MateStrysewske** [comment]
Shouldn't be investigated before #4938 has been done

**Comment by bernardsilenou** [comment]
@DavidBaldsiefen 
- The coordinates initially linked to the address were wrong before calling the method.
- The reason that it would be good to have the feature limited to only persons in the the filter is because in a national instance with millions of people, this may take long and affect performance.

**Comment by DavidBaldsiefen** [comment]
> @DavidBaldsiefen 
> - The coordinates initially linked to the address were wrong before calling the method.
> - The reason that it would be good to have the feature limited to only persons in the the filter is because in a national instance with millions of people, this may take long and affect performance.

That should be fixed with #4938 , which would make this issue a duplicate. @MateStrysewske

**Comment by MateStrysewske** [comment]
@bernardsilenou Is this still happening?

**Comment by bernardsilenou** [comment]
@MateStrysewske  I did a test on test1401 and cound still reproduce the result.
The image has information about the person: uuid, name, addresss, ...
![image](https://user-images.githubusercontent.com/12996724/116910210-90621500-ac45-11eb-8d93-aad12e6760ed.png)

**Comment by bernardsilenou** [comment]
My goal is to replace the right  coordinate of this one person.
- I opened the case person and gave a valid address + generate coordinate. 
-  After generation the coordinate, I then added an error to both lat and long, eg from 10.5 to 11.5, saved the person  form and return to person directory
- search for this person and "set missing coordinate"
- System tool about a minute and returned error.
-  After 4 trial, I got a successful message. However, the old coordinates with error (Inhoffenstraße 7, 38124, Braunschweig: 51.21099, 11.53128) were not set to the right coordinates (52.21099, 10.53128). After many trials, the right coordonates got updated, when I repeated all over again and it breaks. I can not tell what is is the cause, I can can say it that the feature is not stable.

- I think the current solution to update all coordinates of persons in the jurisdiction of the user can be changed to only update this data for persons returned by the filter. Persons that the user has access to but do not filter for them should not be updated. This force the user to wait
-  Event if there is no bug in the code, the method wold still crash in an instance with millions of entities and many of them having non or wrong coordinates.
